database/wiiu/get_user_info_by_pid.go

```diff
@@ -1,44 +1,36 @@
 package database_wiiu
 
 import (
-	"context"
 	"encoding/base64"
 
-	"github.com/PretendoNetwork/friends-secure/database"
-	"github.com/PretendoNetwork/friends-secure/globals"
+	"github.com/PretendoNetwork/friends/globals"
 	"github.com/PretendoNetwork/nex-go"
 	friends_wiiu_types "github.com/PretendoNetwork/nex-protocols-go/friends-wiiu/types"
-	"go.mongodb.org/mongo-driver/bson"
-	"go.mongodb.org/mongo-driver/mongo"
-	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
 func GetUserInfoByPID(pid uint32) *friends_wiiu_types.PrincipalBasicInfo {
-	var result bson.M
+	info := friends_wiiu_types.NewPrincipalBasicInfo()
 
-	err := database.MongoCollection.FindOne(context.TODO(), bson.D{{Key: "pid", Value: pid}}, options.FindOne()).Decode(&result)
+	userData, err := globals.GetUserData(pid)
```

Comment:
Since this uses grpc now, this really shouldn't be in the database package

It should also return an error

Reply:
> Since this uses grpc now, this really shouldn't be in the database package

Not sure what you mean here?

Reply:
The database packages are for functions which directly interact with the database

`GetUserData` is a gRPC call, this no longer interacts with the database directly